lib/custom_widgets/fav_button_counter_column.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+import '../ui_constants.dart';
+
+class FavButtonCounterColumn extends StatefulWidget {
```

Comment:
For the naming is not necessary to add the type of the widget that you are going to return.
You can change this name to something like FavoriteButtonCounter or similar

Reply:
done